src/Sulu/Bundle/MediaBundle/Api/Collection.php

```diff
@@ -40,19 +41,73 @@
  *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999, "types" = "{types}" }
  *      )
  * )
+ * @Relation(
+ *      "self",
+ *      href = @Route(
+ *          "get_collection",
+ *          parameters = { "id" = "expr(object.getId())" }
+ *      )
+ * )
+ * @Relation(
+ *      "admin",
+ *      href ="expr('media/collections/edit:'~object.getId()~'/files')"
```

Comment:
@danrot we have discussed this a week ago and found no fast solution for that perhaps i should add an issue for that? we could change that very fast if @3rfan has implemented this functionality! but i think we should really merge this PR because its big and an important feature which should be really tested before the next release!